tests/serialization/test_dag_serialization.py

```diff
@@ -446,6 +447,26 @@ def validate_deserialized_task(
                 datetime(2019, 8, 1, tzinfo=timezone.utc),
             ),
             (pendulum.datetime(2019, 8, 1, tz='UTC'), None, pendulum.datetime(2019, 8, 1, tz='UTC')),
+            (
+                pendulum.parse("2019-08-01T00:00:00.000+00:00"),
+                None,
+                pendulum.parse("2019-08-01T00:00:00.000+00:00"),
+            ),
+            (
+                pendulum.parse("2019-08-01T00:00:00.000+01:30"),
+                None,
+                pendulum.parse("2019-08-01T00:00:00.000+01:30"),
+            ),
+            (
+                pendulum.datetime(2019, 8, 1, tz='Europe/Stockholm'),
+                None,
+                pendulum.datetime(2019, 8, 1, tz='Europe/Stockholm'),
+            ),
+            (
+                pendulum.datetime(2019, 8, 1, tz=FixedTimezone(3600)),
+                None,
+                pendulum.datetime(2019, 8, 1, tz=FixedTimezone(3600)),
+            ),
         ]
```

Comment:
@ashb , any of this added cases with offset timezones "+01:00", `FixedTimezone(3600)`, etc will raise an `InvalidTimezone` exception in the  `SerializedDAG._deserialize_timezone()` at the  `pendulum.timezone(xxx)`. 

Those timezones serialize to  the timezone name which is `+01:00`,etc and `pendulum.timezone('+01:00')` raises `InvalidTimezone` because that method is meant to resolve "named" timezones only (so only works with `UTC`, `Europe/Stockholm`, `EST`, etc  )

Reply:
So this works:

```
In [13]: pendulum.timezone(3600)                                                                                                                       
Out[13]: Timezone('+01:00')
```

Maybe we need to change the serialization format to accept string or int -- I think that might be less hacky.

Reply:
Maybe coupled with `tz_name in pendulum.tz.timezones`?

Reply:
I can try but I really that will add some very pendulum-specific logic. 

I mean, it will involve checking if this is a `pendulum.tz.timezone.FixedTimezone` and serializing to int if so, there is no "standard" good way of checking if a given `datetime.tzinfo` is a simple timezone (fixed offset only) or a full fledged timezone (with arbitrarily complex dst rules, etc). 

In principle the user can provide a `datetime.tzinfo` of it's own with name `RubenTZ` and that will fail the `pendulum.timezone('RubenTZ')` but doesn't mean that we can just serialize to a fixedoffset (RubenTZ is much more complex) 

What I'm trying to say is that is going to endup being very hackish anyway, but in the serializing part instead of the deserializing part.

Reply:
> Maybe coupled with tz_name in pendulum.tz.timezones?

I'm not sure what you mean , but if you mean to use `.tzname()` instead of `.name` I can already tell you that is even more brittle.